### ISSUE TITLE
Failed to establish connection

### ISSUE THREAD

**Issue by mrxish0t551** [wilschmidtt/YouTube-viewBot#3]
[~] Beginning iteration [1] of [1]

[~] Please ignore [WDM] messages

[~] Connecting to proxy. Please wait ...

[~] Failed to establish connection. Moving to next iteration ...

[~] [1] iterations failed due to failed connection

i know this is a bit outdated but i would still love to use this tool. could you please help me out i got nordvpn installed.

**Comment by wilschmidtt** [comment]
Of course, would you be able to give me a couple days to look at it? I'm out of town until Tuesday but I can check it out once I'm back

**Comment by mrxish0t551** [comment]
Ah sure it's alright i've fixed it by adding the .py file to the nordvpn path. Thanks for your contribution.

**Comment by wilschmidtt** [comment]
Glad you were able to figure it out!
